gui/src-tauri/src/main.rs

```diff
@@ -38,22 +39,53 @@ fn update_window_state(
 	Ok(())
 }
 
-fn main() {
+#[tauri::command]
+fn logging(msg: String) {
+	log::info!(target: "webview", "[webview] {}", msg)
```

Comment:
I think its an antipattern to include `[webview]` in the log text. Its generally the responsibility of the logger implementation to report the target in the logs.

Reply:
It shows the crate name, which is `[slimevr]` :c